test/integration/master/test_build_artifact.py

```diff
@@ -12,12 +13,21 @@
 class TestBuildArtifact(BaseIntegrationTestCase):
     @classmethod
     def setUpClass(cls):
+        # For timing file test
         cls._timing_file_fd, cls._timing_file_path = mkstemp()
 
+        # For parsing subjob/atom ids from build artifact test.
+        cls._artifact_directory_path = mkdtemp()
```

Comment:
Probably doesn't matter much but we could use a `tempfile.TemporaryDirectory` here. I think that cleans itself up, even if the test didn't make it to `tearDownClass()` for some reason.